### PR TITLE
Split port_groups migration to prevent issues

### PR DESCRIPTION
**Pull request by murrant**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: database/migrations/2020_12_14_091314_create_port_groups_table.php [hunk 1]
```diff
@@ -21,8 +21,6 @@ public function up()
             $table->unsignedInteger('port_group_id')->unsigned()->index();
             $table->unsignedInteger('port_id')->unsigned()->index();
             $table->primary(['port_group_id', 'port_id']);
-            $table->foreign('port_group_id')->references('id')->on('port_groups')->onDelete('CASCADE');
-            $table->foreign('port_id')->references('port_id')->on('ports')->onDelete('CASCADE');
         });
```
Comment (Jellyfrog) [inline database/migrations/2020_12_14_091314_create_port_groups_table.php:24]: Best practice is to have one table migration per migration file, I read somewhere Taylor said that, and it makes sense either way

Comment (murrant) [reply database/migrations/2020_12_14_091314_create_port_groups_table.php:24]: true, I can split it out into 4 migrations.

File: database/migrations/2020_12_14_091314_create_port_group_port_table.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+class CreatePortGroupsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        if (Schema::hasTable('port_group_port')) {
```
Comment (Jellyfrog) [inline database/migrations/2020_12_14_091314_create_port_group_port_table.php:15]: If not?

File: database/migrations/2020_12_14_091314_create_port_group_port_table.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreatePortGroupsTable extends Migration
```
Comment (Jellyfrog) [inline database/migrations/2020_12_14_091314_create_port_group_port_table.php:6]:
```suggestion
class CreatePortGroupPortsTable extends Migration
```

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**